backend/main/src/User/Domain/User.ts

```diff
@@ -31,20 +25,28 @@ export class User extends AggregateRoot<string, IUserProps> {
 		return this.props.name;
 	}
 
-	get isAdmin(): boolean {
```

Comment:
This change makes codes for checking user types unnecessary.
It indicates the following codes and some test codes.

> backend/main/src/User/UseCases/UpdateUserUseCase/UpdateUserNameUseCase.ts: 44-